src/bank.rs

```diff
@@ -1011,6 +1019,18 @@ impl Bank {
         let execution_elapsed = now.elapsed();
         let now = Instant::now();
         self.store_accounts(txs, &executed, &loaded_accounts);
+
+        // Send notifications
+        {
+            let accounts = self.accounts.read().unwrap();
```

Comment:
would it be harmful to just always send the resulting value, i.e. assume the transaction caused a change?

Reply:
one reason I can think of for doing the simplest thing here is that rollbacks cause the same notifications, and we're not doing any diffing in that code

Reply:
finally, do we have a test for the "don't notify unless changed" case?

Reply:
I've been meaning to check in with @mvines on this one: what do you think about notifying on the assumption that the transaction caused a change?
It would certainly make the code a lot simpler.

Reply:
A rollback event I think should be signaled differently regardless: "You've just been wormholed to a different universe, you may not even exist here. Cold reboot the front-end."

The proposal is to notify on all accounts that a transaction instruction *could* have written to, even if it didnt?  This won't break any of our existing demos (and if so good, because they're buggy and we should fix), so beyond unnecessary notifications that perhaps we can optimize away elsewhere I don't see any issue with removing the diffing.